README.md

```diff
@@ -29,3 +30,64 @@ To test if it is installed correctly,
 ```julia
 pkg> test ArchGDAL
 ```
+
+## Getting Involved
+
+### Community
+
+This package will not be possible without https://julialang.org/, https://github.com/OSGeo/gdal and https://github.com/JuliaGeo/GDAL.jl. They are maintained by https://julialang.org/community/, https://www.osgeo.org/ and https://juliageo.org/. In case of any contention for support and involvement, we encourage participation and contributions to those projects and communities over this package.
+
+### Style Guide
+
+ArchGDAL.jl uses [JuliaFormatter.jl](https://github.com/domluna/JuliaFormatter.jl) as
+an autoformatting tool, and uses the options in [`.JuliaFormatter.toml`](https://github.com/yeesian/ArchGDAL.jl/blob/master/.JuliaFormatter.toml).
+
+To format code, `cd` to the ArchGDAL.jl directory, then run:
+```julia
+] add JuliaFormatter@0.13.10
+using JuliaFormatter
+format("src")
+```
+
+A continuous integration check verifies that all PRs made to ArchGDAL.jl have passed the formatter.
+
+### Dependencies
+To manage the dependencies of this package, we work with [environments](https://pkgdocs.julialang.org/v1.6/environments/):
+
+1. Navigate to the directory corresponding to the package:
+
+```julia
+shell> cd /Users/yeesian/.julia/dev/ArchGDAL
+/Users/yeesian/.julia/dev/ArchGDAL
+```
+
+2. Activate the environment corresponding to `Project.toml`):
+
+```julia
+(@v1.6) pkg> activate .
+  Activating environment at `~/.julia/environments/v1.6/Project.toml`
+```
+
+3. Manage the dependencies using Pkg in https://pkgdocs.julialang.org/v1.6/managing-packages/, e.g.
+
+```julia
+(ArchGDAL) pkg> st
+     Project ArchGDAL v0.6.0
+      Status `~/.julia/dev/ArchGDAL/Project.toml`
+  [3c3547ce] DiskArrays
+  [add2ef01] GDAL
+  [68eda718] GeoFormatTypes
+  [cf35fbd7] GeoInterface
+  [bd369af6] Tables
+  [ade2ca70] Dates
+
+(ArchGDAL) pkg> add CEnum
+   Resolving package versions...
+    Updating `~/.julia/dev/ArchGDAL/Project.toml`
+  [fa961155] + CEnum v0.4.1
+  [3c3547ce] + DiskArrays v0.2.7
+  [add2ef01] + GDAL v1.2.1
+  [68eda718] + GeoFormatTypes v0.3.0
+  [cf35fbd7] + GeoInterface v0.5.5
+  [bd369af6] + Tables v1.4.2
+```
```

Comment:
We should add  a section about the compat entries. 
These are needed, so that Julia can resolve the package versions and it is good practice to have compate entries for all dependencies.

```suggestion
```4. Add a compat entry into Project.toml so that julia can resolve the versions
```     Open the Project.toml file in a texteditor and add in the [compat] section the line:
        CEnum = "0.4"

```

Reply:
Thank you!